manifests/Quazar Conundrum.yaml

```
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/tree/main
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galxy spanning threat, changing hegemony and new factions rising to power.
+url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
```

this needs to be a direct link to a zip file, such as
```suggestion
url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/Major_ReleaseV015.zip
```